docs/competency-api/behavior.md

```diff
@@ -4,4 +4,58 @@ title: Behavior
 nav_order: 3
 parent: Competency API
 ---
-# Behavior
\ No newline at end of file
+# Behavior
+
+## Update Behavior
+
+### Description 
+
+#### patch:
+```http
+https://api.competency-constructor.clark.center/competencies/:competencyId/behavior
+```
+
+### Headers
+
+```json
+{
+    Bearer: "Bearer Example.bearer.token"
+}
+```
+
+### Request
+
+| Name | Required | Type | Description |
+|:----:|:-----:|:----:|-----|
+| task | true | string | The task to be carried out by the student |
+| details | false | string | Any details related to how the task should be completed |
+| work_role | false | string | Work role(s) in which this task could be performed |
```

Comment:
`work_role` is actually going to be an object id and only one can be sent i believe. Correct me if im wrong @mfranz1